docs/source/config.rst

```diff
@@ -25,6 +25,7 @@ Server Configuration
 ~~~~~~~~~~~~~~~~~~
 - ``DAGU_HOST`` (``127.0.0.1``): Server binding host
 - ``DAGU_PORT`` (``8080``): Server binding port
+- ``DAGU_BASE_PATH`` (``""``): Base path to serve the application
```

Comment:
While the ticket says `BAUSE_URL` I feel `PATH` is more accurate. 

URL implies including the origin as well, and we're only supplying the path of the URL. 

I'm happy to change this if desired - it just seemed better to be accurate here. A note: `ApiBaseURL` should also be called path but that seemed out of scope for this work.

Reply:
Yes, 100% agreed. Thank you!